tests/wdio/matchScreenshotAndTestStandards-spec.js

```diff
@@ -0,0 +1,38 @@
+/* global describe, before, browser, it, expect, Terra */
```

Comment:
If we are using our eslint-configuration, we shouldn't need to add these

Reply:
Yeah it's on my list to figure out why that's not working.